.github/workflows/stale.yml

```diff
@@ -0,0 +1,19 @@
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/stale@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Does anybody know how this works?

Reply:
Token ~ Username+Password
You can create/remove Tokens, give them different rights, add experience date... That makes them usefull in combination with APIs. It is important to keep them secure. Therefor you store them as a secret. The GitHub token is already predefined for the actions.

Reply:
OK